node_modules/gh-orgunit/lib/internal/import.js

```diff
@@ -31,19 +31,30 @@ var OrgUnitDAO = require('gh-orgunit/lib/internal/dao');
 /**
  * Import a set of organisational units, event series and events into an organisational unit
  *
- * @param  {Context}        ctx                     Standard context containing the current user and the current app
- * @param  {Number}         id                      The id of the organisational unit into which to import the data
- * @param  {Object}         data                    The organisational units, event series and events to import
- * @param  {Boolean}        deleteMissing           Whether to delete items from the database that aren't in the import data
- * @param  {Function}       callback                Standard callback function
- * @param  {Object}         callback.err            An error object, if any
+ * @param  {Context}        ctx                         Standard context containing the current user and the current app
+ * @param  {Number}         id                          The id of the organisational unit into which to import the data
+ * @param  {Object}         data                        The organisational units, event series and events to import
+ * @param  {Object}         options                     A set of options
+ * @param  {Boolean}        options.deleteMissing       Whether to delete items from the database that aren't in the import data. Defaults to `false`
+ * @param  {Boolean}        options.checkExternalId     Whether the provided `data` should be checked for valid external ids. Defaults to `true`
+ * @param  {Function}       callback                    Standard callback function
+ * @param  {Object}         callback.err                An error object, if any
  */
-var importOrgUnit = module.exports.importOrgUnit = function(ctx, id, data, deleteMissing, callback) {
+var importOrgUnit = module.exports.importOrgUnit = function(ctx, id, data, options, callback) {
+    options = options || {};
+    options.deleteMissing = options.deleteMissing || false;
+    if (options.checkExternalId !== false) {
+        options.checkExternalId = true;
+    }
```

Comment:
Or rather, `_.ghOptional` will take care of initializing `options` and adding default values:

```javascript
options = _.ghOptional(options, {
    'deleteMissing': false,
    'checkExternalId': true
});
```